builtin/logical/database/backend.go

```diff
@@ -55,31 +74,37 @@ func Backend(conf *logical.BackendConfig) *databaseBackend {
 		Help: strings.TrimSpace(backendHelp),
 
 		PathsSpecial: &logical.Paths{
+			LocalStorage: []string{
+				framework.WALPrefix,
+			},
 			SealWrapStorage: []string{
 				"config/*",
+				"static-role/*",
 			},
 		},
-
-		Paths: []*framework.Path{
-			pathListPluginConnection(&b),
-			pathConfigurePluginConnection(&b),
+		Paths: framework.PathAppend(
+			[]*framework.Path{
```

Comment:
What is the significance of having a subset of paths here, vs. just a flat list?

Reply:
This was changed so that this list didn't contain entries like so:

```
pathListRoles
pathListStaticRoles
pathRoles
pathStaticRoles
pathCredsCreate
pathStaticCredsCreate
```

instead, we keep the original `pathListRoles` et.al. and then those methods return `[]*framework.Path` (one for "normal", another for "static") instead of the individual `*framework.Path`.